### PR TITLE
Bump libocr => fd3cab206b2ca3b7ff207996b95673b2d6303ec4

### PR DESCRIPTION
**Pull request by samsondav**
Needed for reducing default poll time

### CODE REVIEW

**Review by jmank88 — DISMISSED** [review]

**Review by jmank88 — APPROVED** [review]

**Comment by augustbleeds** [comment]
Need to either cherry-pick the commit here https://github.com/smartcontractkit/chainlink/pull/12905/commits/a7629135146ef42a57804e976d13ffcaff807c8b or make the corresponding changes in the plugins/ directory

**Review by augustbleeds — CHANGES_REQUESTED** [review]
Need to either cherry-pick the commit here https://github.com/smartcontractkit/chainlink/commit/a7629135146ef42a57804e976d13ffcaff807c8b or make the corresponding changes in the `plugins/` directory or else there's risk of the plugin not being properly initialized

File: .changeset/stale-terms-march.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"chainlink": patch
+---
+
+Bump libocr => fd3cab206b2ca3b7ff207996b95673b2d6303ec4
```
Comment (jmank88) [inline .changeset/stale-terms-march.md:5]: Shouldn't we mention the new feature?

Comment (samsondav) [reply .changeset/stale-terms-march.md:5]: What is the new feature?

Comment (augustbleeds) [reply .changeset/stale-terms-march.md:5]: It's not being used at the moment. But if you want to say it you could say "introduces the ability to report GasPriceSubunits in the median plugin - currently value is ignored"

Comment (jmank88) [reply .changeset/stale-terms-march.md:5]: I didn't realize it wasn't connected yet. No need to mention then I guess

Comment (jmank88) [reply .changeset/stale-terms-march.md:5]: It sounds like something with poll time changed though?

**Review by jmank88 — APPROVED** [review]

**Pull request merged**